INSTALL.md

```diff
@@ -20,78 +20,87 @@ The system requirements depend on how you install UDOIT.  If you use Docker, the
 
 ## .ENV Setup
-UDOIT uses a `.env` file for storing configuration variables. To create it:
+UDOIT stores configuration variables in a `.env` file. To create it:
 
-1. Copy the file `.env.example` to `.env` by running
+1. Inside the UDOIT directory, run the following command in shell:
 ```
 cp .env.example .env
 ```
-2. Leave `APP_ENV` set to `prod`
-> If you are setting up a development environment, set this to `dev` and follow the steps in [Installing Composer Dependencies](#installing-composer-dependencies) without the `--no-dev` flag to obtain all of the development packages.
-3. Add your database information to the `DATABASE_URL` variable.  (The default value of `mysql://root:root@db:3306/udoit3` is suitable for running it on your local computer using Docker.)
-4. Modify the `BASE_URL` to match the URL of your instance of UDOIT.  (The default value of `http://127.0.0.1:8000/udoit3` is suitable for running it on your local computer using Docker.)
-5. Modify the `WEBPACK_PUBLIC_PATH` to match the `BASE_URL` you set.  (The default value of `/udoit3/build` is suitable for running it on your local computer usind Docker.  Example:  If your `BASE_URL` is set to `http://127.0.0.1:8000`, your `WEBPACK_PUBLIC_PATH` should be `/build`.)
-6. Set `APP_LMS` to the name of your LMS.
-   * `canvas` if you are using the Canvas LMS.
-   * `d2l` if you are using the D2l Brightspace LMS.
-7. If you are using UDOIT with a self-hosted instance of Canvas, you can add the `JWK_BASE_URL` variable and set it to the URL of your instance of Canvas. (Example: `JWK_BASE_URL="https://canvas.dev.myschool.edu"`)
-8. (Optional) You can change the default language for your entire UDOIT instance by adding the `DEFAULT_LANG` variable. Currently supported languages are English (`en`) and Spanish (`es`).
-
-## Docker
-We provide a fast and simple way of setting up a local UDOIT instance through the use of Docker containers.
-
-### 1. Install Docker
-To set up the docker containers, you must first install [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/).
-
-### 2. Build the Containers
-If you prefer to build the containers yourself, or you are actively developing UDOIT and need to rebuild the containers to test your cahnges, run the following command from within the UDOIT directory:
-
+This command copies the `.env.example` into `.env`, creating the `.env` file in the process if it does not exist.
+
+2. Open `.env` with a text editor and make the necessary changes to the following variables:
+	- `APP_ENV`: If you are setting up a development environment, change this to `dev` and follow the steps in [Installing Composer Dependencies](#installing-composer-dependencies) without the `--no-dev` flag to obtain all of the development packages. Otherwise, leave it as `prod`.
+	- `DATABASE_URL`: If you are hosting UDOIT on Docker or your local machine, leave it as it is. Otherwise, change it to your database URL.
```

Comment:
```suggestion
	- `DATABASE_URL`: If you are hosting UDOIT on Docker locally, leave it as it is. Otherwise, change it to your database URL.
```
This only works if Docker is running on your local machine, not either or. Same suggestion for the following two lines, but I'm good with any other similar changes!

Reply:
If UDOIT is being hosted locally, doesn't that also mean the database is available too? Or are we seeing the database as a separate piece?

Reply:
They should be separate, though I definitely misread what your comment was trying to say. Will make a small change and work on pushing this up to `dev`!